spec/index.bs

```diff
@@ -661,6 +661,8 @@ take advantage of those capabilities effectively. The general mitigation approac
 normalizing or binning potentially identifying information and enforcing uniform behavior where
 possible.
 
+A user agent must not reveal more than 32 distinguishable configurations or buckets.
```

Comment:
This is in a non-normative section. Maybe this can be in the optional-capabilities section? (keep a note here too though)